data_management_basics/data_management_basics.md

```diff
@@ -0,0 +1,295 @@
+
+### Quiz: Stakeholder Responsibilities
+
+Which of the following is not a responsibility of the principal investigator of a research project?
+
+[( )] Planning for the responsible and ethical use of data
+[(X)] Handling disclosures of scientific data generated from the study, once that data is submitted to a central database
+[( )] Ensuring adherence to research protocols and to applicable laws and regulations
+[( )] Being the ultimate responsible party for statistical and procedural rigor in the scientific work
+****
+<div class = "answer">
+
+All of these are the responsibility of the principal investigator, except for handling disclosures of scientific data generated from the study, once that data is submitted to a central database.  Once data is submitted to a repository, it is generally the repository who handles data disclosures. Sometimes this is accomplished through a scientific committee, sometimes through a simple application or attestation, and occasionally by way of an unregulated download, in the case of non-sensitive data of general public interest.˚
+
+</div>
+****
+
+## Quiz: Why RDM?
+
+Why is it important to practice proper research data management? Select all that apply.
+
+[[X]] Research data management helps researchers create higher quality research output.
+[[X]] Research data management supports the long-term preservation of data for future use.
+[[ ]] Few publishers have standards for data associated with manuscripts, so researchers have to apply their own rigor.
+[[X]] Funding agencies supporting the research require research data management.
+[[ ]] Research data management has recently appeared as a new practice, and performing this work shows that researchers are up-to-date.
+[[?]] There are three correct answers!
+****
+<div class = "answer">
+
+Research data management is important for multiple reasons, among which we highlighted three in this question:
+
+* Research data management helps researchers create higher quality research output.
+* Research data management supports the long-term preservation of data for future use.
+* Funding agencies supporting the research require research data management.
+
+It's false that publishers don't apply their own standards.  Publishers have an important role to play in setting high standards for researchers who submit manuscripts for consideration.  It's also false that research data management is a new practice.  While it's true that RDM has gotten much more attention lately, research data management has been taking place since the first researchers developed the scientific method, and it's changed over time to adapt to new circumstances and requirements.
+
+</div>
+***
+
+## Additional Resources
+
+The [data sharing policies and helpful information](https://sharing.nih.gov/data-management-and-sharing-policy
```

Comment:
i think the code here is a little faulty. When I went into the liascript version of the code, the full link was visible and not hyperlinked

Reply:
Whoops I'll fix